packages/SketchMorph2-Core.package/M2Backend.class/instance/setBrushAndEraserSizeTo..st

```diff
@@ -0,0 +1,6 @@
+accessing
+setBrushAndEraserSizeTo: aNumber
```

Comment:
Maybe we should make an interface that takes both sizes as different numbers, that gets called by this method?